game_event/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from game_event_player.models import GameEventPlayer
+from court.models import Court
+
+
+class BallGame(models.TextChoices):
+    Soccer = 'Soccer', 'Soccer'
+    Basketball = 'Basketball', 'Basketball'
+    Volleyball = 'Volleyball', 'Volleyball'
+    Baseball = 'Baseball', 'Baseball'
+    Tennis = 'Tennis', 'Tennis'
+    Rugby = 'Rugby', 'Rugby'
+    Golf = 'Golf', 'Golf'
+    Cricket = 'Cricket', 'Cricket'
+    Handball = 'Handball', 'Handball'
+
+
+class Rating(models.IntegerChoices):
+    One = 1
+    Two = 2
+    Three = 3
+    Four = 4
+    Five = 5
+    Six = 6
+    Seven = 7
+    Eight = 8
+    Nine = 9
+    Ten = 10
+
+
+class GameEvent(models.Model):
+    id = models.IntegerField(primary_key=True)
+    time = models.DateTimeField(null=False)
+    level_of_game = models.IntegerField(choices=Rating.choices, blank=True)
+    min_number_of_players = models.PositiveIntegerField()
```

Comment:
Please consider removing the blank=True, also according to the django documentation null/blank Defaults are set to False, so you can remove the attribute completly if you like.

Reply:
Maybe we can just set the game event level to be the lowest level possible for this case. That would achieve your goal.

Reply:
@amitKaplansky if this is something that blocking you, I suggest proceeding with the easiest approach to implement while opening an issue to have it figured out.

Reply:
Should I add to the class a Meta class with unique_together = ('ball_responsible_player', 'court_id')? @yftacherzog @Yuval-Vino

Reply:
I don't know. Up to your design decisions.

I really really suggest that you proceed with the easiest way forward at this point and move to adding tests so the PR can be properly reviewed. You can (should) create an issue for any improvements you consider adding.

You now have 2 workdays to get this merged, and it's barely enough time for typical review cycles, and you still don't have the tests here, meaning the code is not complete, and you need to leave enough time to handle rebase issues etc.

At demo time, it will be much (much) more important to have everything merged (it must have proper tests to be merged) comparing to having some missing attributes.